shell/ios/ViewController.mm

```diff
@@ -138,7 +138,7 @@ - (void)initRenderSessionController {
     IGL_ASSERT(platformDevice);
     return igl::SurfaceTextures{
         .color = platformDevice->createTextureFromNativeDrawable((CAEAGLLayer*)layer_, nullptr),
-        .depth = platformDevice->createTextureFromNativeDepth((CAEAGLLayer*)layer_, nullptr),
+        .depth = platformDevice->createTextureFromNativeDepth((CAEAGLLayer*)layer_, igl::TextureFormat::S8_UInt_Z24_UNorm, nullptr),
```

Comment:
Same. Use the old value ` TextureFormat::Z_UNorm24`.

Reply:
The ios old value is Z_UNorm16. 
Done.